### ISSUE TITLE
Include community roles in person dashboard view

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy#2264]
The person dashboard view should include a list of all community roles with start/end dates as well as other associated data (award, membership, url, and custom fields).

**Comment by maneesha** [comment]
@pbanaszkiewicz @elichad is there any other information needed to implement this feature?

**Comment by pbanaszkiewicz** [comment]
@maneesha do you mean "Your Profile" (instructor dashboard) view?

**Comment by maneesha** [comment]
Yes, I mean this view: https://amy.carpentries.org/dashboard/instructor/
